modules/keepalived-limitation-for-ipi.adoc

```diff
@@ -0,0 +1,48 @@
+[discrete]
+[id="keepalived-limitation-for-ipi_{context}"]
+= Identifying pre-existing virtual routers
+
+With installer-provisioned infrastructure (IPI) installations, the Virtual IPs are managed using multicast (VRRPv2 or VRRPv3), and there is a limitation of 255 unique virtual routers per multicast domain. Each VIP has a virtual router id associated with it based on the cluster name. Duplication of these ids can result in a conflict, preventing the Virtual IPs from being assigned to the node. Therefore, if you have pre-existing virtual routers using the standard IPv4 or IPv6 multicast groups, you should check to see which VIPs the installation will choose to avoid conflicts.
+
+To discover the VIPs that will be chosen, you can use the runtimecfg utility.
+
+.Procedure
+
+. Run the runtimecfg utility with a domain name, in this example `cnf10`:
```

Comment:
@eslutsky @janosdebugs should this be the metadata.name or the metadata.name +baseDomain or something else?

see the references here https://docs.openshift.com/container-platform/4.8/installing/installing_rhv/installing-rhv-customizations.html#installation-configuration-parameters-required_installing-rhv-customizations

Reply:
Not sure, unfortunately.

Reply:
Ok, question here: why is `cnf10` the example? Where does this come from? I tried to search for it, and this is confusing.

Reply:
The baremetal example I used when raising the BZ[1] had this set, it is referring to the Cluster Name set in the installer or install-config.yaml for the installer.


[1] https://github.com/openshift/installer/blob/master/docs/user/metal/install_ipi.md


[1] https://github.com/openshift/installer/blob/master/docs/user/metal/install_ipi.md

Reply:
Would it make more sense to say "the cluster name" instead "domain name"?

Reply:
Yes I'd say that makes more sense and doesn't cause any confusion with the "baseDomain" setting in the installer.

Reply:
Done.